tests/test_interface.rb

```diff
@@ -1350,20 +1325,18 @@ def test_interface_ipv4_all_interfaces
     rescue Minitest::Assertion
       # clean up before failing
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
       raise
     end
   end
 
   def test_interface_vrf_default
-    config('interface loopback1', 'vrf member foo')
     interface = Interface.new('loopback1')
     interface.vrf = interface.default_vrf
     assert_equal(DEFAULT_IF_VRF, interface.vrf)
   end
 
   def test_interface_vrf_empty
-    config('interface loopback1', 'vrf member foo')
```

Comment:
With this change to both test cases, you're no longer testing the case where an interface is in a VRF and you're setting it back to the default, right? That seems like a gap.